### DISCUSSION TITLE
How to reuse template on other sites.

### DISCUSSION THREAD

**Discussion by N6REJ** [templaza/astroid-framework#490] [Q&A] [answered]
Once I've built my site locally and have a basic template I want to use on other sites, how can I export Asteroid that I've built up as an installable template on other sites.

**Comment by paurray** [comment]
Maybe this helps: https://docs.astroidframe.work/more/export-import

**Comment by aleonha1** [comment]
Gilt dieser Hinweis auch Framework 3.0.2
ich kann diesen Menue Punkt nicht finden?

**Comment by paurray** [comment, marked as answer]
Yes it has moved in Astroid 3 +
In the Template Styles Section/Template/Template Options
Look down in the ver bottom left for “Preset Profiles” in there is a “Save/Import” option

**Comment by aleonha1** [comment]
Ich kann dies Funktion in meinem Template nicht finden?
![TP_gym_Astroid_3 0 2](https://github.com/templaza/astroid-framework/assets/67064507/71bcb309-4668-4ed4-8481-37fdd4bba8f3)

**Comment by aleonha1** [comment]
Alles klar ich habe den Vorgang gefunden
